json.Unmarshall error while using sdp from browser for pub-from-browser example

<!--
  Come join our community on Slack https://pion.ly/slack for quicker support and more resources

  We also provide a Knowledge Base https://pion.ly/knowledge-base that covers Pion and general WebRTC
  
  We maintain a FAQ of common issues and questions https://pion.ly/knowledge-base/pion-basics/faq/

 Please answer these questions before submitting your issue. Thanks!
-->

### Your environment.
  - Version: *0dd6b83fa52f9726695a59e4699537da9e6becbb*
  - OS: *Arch Linux*
  - Browser: *Google Chrome Version 81.0.4044.129 (Official Build) (64-bit)*

### What did you do?
I was following the examples code for **pub-from-browser**

I copied the **base64 sdp** from from top text are of **jsfiddle** in expample **README.md**

Pasted sdp to sdp.txt
then,

`cat sdp.txt | go run pub_from_browser/main.go`

<!--
If possible, provide an example for reproducing the error.
A complete runnable program is good.
-->

### What did you expect?
I expected to get an another sdp to paste in jsfiddle and start the session

### What happened?
Then I get following error traceback:

```
[wrocket@wrocket-pc pub-from-browser]$ cat sdp.txt | go run main.go

panic: illegal base64 data at input byte 1

goroutine 1 [running]:
github.com/pion/ion-sfu/examples/internal/signal.Decode(0xc000016a00, 0x2124, 0x94e3a0, 0xc000074af0)
	/home/wrocket/git/ion-sfu/examples/internal/signal/signal.go:74 +0xbf
main.main()
	/home/wrocket/git/ion-sfu/examples/pub-from-browser/main.go:30 +0x20d
exit status 2
```


I found the **sdp value in json is string** and `Sdp` in `SDPDescription` struct  in proto3 is []byte.
And because of it we are having issue in `json.Unmarshall `

--- a/examples/pub-from-browser/main.go
+++ b/examples/pub-from-browser/main.go
@@ -10,6 +10,7 @@ import (
 
 	"github.com/pion/ion-sfu/examples/internal/signal"
 	sfu "github.com/pion/ion-sfu/pkg/proto"
+	"github.com/pion/webrtc/v2"
 	"google.golang.org/grpc"
 )
 
@@ -26,7 +27,7 @@ func main() {
 	defer conn.Close()
 	c := sfu.NewSFUClient(conn)
 
-	pubOffer := sfu.SessionDescription{}
+	pubOffer := webrtc.SessionDescription{}
 	signal.Decode(signal.MustReadStdin(), &pubOffer)
 
 	if err != nil {
@@ -44,7 +45,10 @@ func main() {
 		Rid: "default",
 		Payload: &sfu.PublishRequest_Connect{
 			Connect: &sfu.Connect{
-				Description: &pubOffer,
+				Description: &sfu.SessionDescription{
+					Type: pubOffer.Type.String(),
+					Sdp:  []byte(pubOffer.SDP),
+				},
 			},
 		},
 	})
@@ -69,7 +73,11 @@ func main() {
 		case *sfu.PublishReply_Connect:
 			// Output the mid and answer in base64 so we can paste it in browser
 			fmt.Printf("\npub mid: %s", res.Mid)
-			fmt.Printf("\npub answer: %s", signal.Encode(payload.Connect.Description))
+			fmt.Printf("\npub answer: %s", signal.Encode(
+				webrtc.SessionDescription{
+					Type: webrtc.SDPTypeAnswer,
+					SDP:  string(payload.Connect.Description.Sdp),
+				}))
 		}
 	}
 }

